### PR TITLE
Added Zero routing to MWSS

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Added Zero routing around employee breakdown questions

Fixes https://github.com/ONSdigital/eq-survey-runner/issues/1499

### CODE REVIEW

**Comment by shahi645** [comment]
Yarn Format

File: data/en/test_sum_equal_or_less_validation_against_total.json [hunk 1]
```diff
@@ -90,4 +90,4 @@
             ]
         }]
     }]
-}
+}
```
Comment (shahi645) [inline data/en/test_sum_equal_or_less_validation_against_total.json:93]: why are these committed?

Comment (jonnyshaw89) [reply data/en/test_sum_equal_or_less_validation_against_total.json:93]: yarn formatted them. can only assume they slipped through before

File: data/en/1_0005.json [hunk 2]
```diff
@@ -1850,6 +2017,54 @@
                         "title": "Five Weekly Pay",
                         "type": "Question"
                     },
+                    {
+                        "type": "Question",
+                        "title": "Five Weekly Pay",
+                        "id": "confirm-five-weekly-pay-gross-pay",
+                        "skip_conditions": [{
+                            "when": [{
+                                "id": "four-five-weekly-gross-pay-answer",
```
Comment (shahi645) [inline data/en/1_0005.json]: delete four

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Review by MIslam92 — APPROVED** [review]

File: data/en/1_0005.json [hunk 3]
```diff
@@ -1850,6 +2017,54 @@
                         "title": "Five Weekly Pay",
                         "type": "Question"
                     },
+                    {
+                        "type": "Question",
+                        "title": "Five Weekly Pay",
+                        "id": "confirm-five-weekly-pay-gross-pay",
+                        "skip_conditions": [{
+                            "when": [{
+                                "id": "five-weekly-gross-pay-answer",
```
Comment (shahi645) [inline data/en/1_0005.json]: My bad, you also need to add a `pay` here:
`five-weekly-pay-gross-pay-answer`

**Review by shahi645 — CHANGES_REQUESTED** [review]
If you want to rebase and squash same time, its pretty much there

File: data/en/1_0005.json [hunk 4]
```diff
@@ -385,6 +380,54 @@
...
+                                        "value": "No",
+                                        "id": "confirm-weekly-pay-gross-pay-answer",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "weekly-pay-gross-pay"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "group": "fortnightly-pay"
```
Comment (dwyeradam) [inline data/en/1_0005.json]: @charlotte-hirst. Although the slide shows route to end is there an argument that we should actually route to the significant change question?

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: data/en/1_0005.json (hunk 4 above)
Comment (shahi645) [reply data/en/1_0005.json]: I was thinking that earlier

**Comment by dwyeradam** [comment]
@charlotte-hirst, @will-madebymoon. Nothing to do with this PR but we could easily fix it here. 
On the **Are these all of the pay patterns used?** question the No option states:
**No, I would like to add another pay pattern**
This is fine if you've not selected a pay pattern but doesn't work if you've selected too many. Should it actually be **No, I would like to change the pay patterns**?

File: data/en/1_0005.json (hunk 4 above)
Comment (charlotte-hirst) [reply data/en/1_0005.json]: it should go to significant changes to explain why its zero

**Comment by charlotte-hirst** [comment]
Yes @dwyeradam  i agree it should be  No, I would like to change the pay patterns?

**Review by dwyeradam — APPROVED** [review]
Without me complaining we would have had incorrect routing and q-codes as well.

**Comment by dwyeradam** [comment]
@shahi645 can you just check you're happy.

**Review by shahi645 — APPROVED** [review]
Just needs that special `Yarn format` but everything else is fine

**Pull request merged**